index.html

```diff
@@ -3423,6 +3427,37 @@ <h1>Speakers</h1>
                             <span>True Global Ventures</span>
                         </div>
                     </div>
+
+                    <div class="col-md-3 col-sm-6 speaker-column">
+                        <div class="speaker">
+                            <div class="image-holder">
+                                <img class="background-image" alt="Meng Weng Wong" src="img/mengwengwong.jpg">
```

Comment:
Please follow the naming convention for images across the project. It should be ```MengWengWong.jpg```

Reply:
ok